engine/installation/linux/docker-ce/ubuntu.md

```diff
@@ -41,6 +41,9 @@ Systems) architectures.
 > **`s390x` limitations**: System Z is only supported on Ubuntu Xenial,
 > Yakkety, and Zesty.
 
+### Locating the .deb files
```

Comment:
Actually isn't this already covered around line 248 (248 with your changes applied)? Now that I look at it in place, your addition pretty much covers the same ground.